docs/victoriatraces/README.md

```diff
@@ -0,0 +1,112 @@
+VictoriaTraces is open source user-friendly database for distributed tracing data 
+from [VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics/).
+
+VictoriaTraces provides the following features:
+- It is resource-efficient and fast. It uses up to [**3.7x less RAM and up to 2.6x less CPU**](https://victoriametrics.com/blog/dev-note-distributed-tracing-with-victorialogs/) than other solutions such as Grafana Tempo.
+- VictoriaTraces' capacity and performance scales linearly with the available resources (CPU, RAM, disk IO, disk space).
+- It accepts trace spans in the popular [OpenTelemetry protocol](https://opentelemetry.io/docs/specs/otel/protocol/)(OTLP).
+- It provides [Jaeger Query Service JSON APIs](https://www.jaegertracing.io/docs/2.6/apis/#internal-http-json) 
+  to integrate with [Grafana](https://grafana.com/docs/grafana/latest/datasources/jaeger/) or [Jaeger Frontend](https://www.jaegertracing.io/docs/2.6/frontend-ui/).
+
+## How does it work
+
+VictoriaTraces is built on top of [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/), which is a log database. 
+It receives trace spans in OTLP format, transforms them into structured logs, and provides [Jaeger Query Service JSON APIs](https://www.jaegertracing.io/docs/2.6/apis/#internal-http-json) for querying.
+
+For detailed data model and example, see: [Key Concepts](https://docs.victoriametrics.com/victoriatraces/keyConcepts).
+
+![How does VictoriaTraces work](how-does-it-work.webp)
+
+Building VictoriaTraces in this way enables it to scale easily and linearly with the available resources, like VictoriaLogs.
+
+## Quick Start
+
+Currently, VictoriaTraces is under actively developing. It provides [binary releases](https://github.com/VictoriaMetrics/VictoriaTraces/releases/latest) and
+docker images ([Docker Hub](https://hub.docker.com/r/victoriametrics/victoria-traces/) and [Quay](https://quay.io/repository/victoriametrics/victoria-traces?tab=tags)). 
```

Comment:
I'd suggest removing these sections for now, as the linked pages currently lead to empty pages. Since we haven't started publishing images yet, it makes sense to add them back once we do.

Reply:
Preserved only build from source.